test/extended/cluster/utils.go

```diff
@@ -106,12 +181,15 @@ func GetTuningSet(tuningSets []TuningSetType, podTuning string) (tuning *TuningS
 	return nil
 }
 
-// Server is the webservice that will syncronize the start and stop of Pods
-func Server(c *PodCount) error {
+// Server is the webservice that will synchronize the start and stop of Pods
+func Server(c *PodCount, Port int, awaitShutdown bool) error {
 	http.HandleFunc("/start", handleStart(startHandler, c))
 	http.HandleFunc("/stop", handleStop(stopHandler, c))
+	if Port <= 0 {
```

Comment:
How could `Port` be < 0?

Reply:
Perhaps because it is a signed int and user defined it as negative?  Most code I've seen in origin uses int for ports.

Reply:
I'd rather it just throw an error in that edge case, but this will be fine, maybe add a log message?